### ISSUE TITLE
Suspected Star Force Simulation Error

### ISSUE THREAD

**Issue by lowrt** [KENNYSOFT/WzComparerR2#20]
Do you have a GMS client? Can you test it?

![97e4d6a644ee34c524cf8e3b5582f247](https://user-images.githubusercontent.com/37591855/71305161-8ea9ff80-240a-11ea-8f4e-4a27176ee60e.jpg)
This is a pet query for GMS players

![1132313](https://user-images.githubusercontent.com/37591855/71305420-b0a58100-240e-11ea-8028-cd6689bd4575.png)
![1113305](https://user-images.githubusercontent.com/37591855/71305421-b0a58100-240e-11ea-8bc1-c20690542451.png)

**Comment by KENNYSOFT** [comment]
Nope I don't have any GMS client. Can you share it?

**Comment by KENNYSOFT** [comment]
Yeah I just found that these items have `onlyUpgrade` properties set.

![image](https://user-images.githubusercontent.com/6624567/90511095-f5004100-e196-11ea-99aa-4f424d0675b8.png)

In KMS, all items with `onlyUpgrade` is not upgradable, so I removed blank stars if a gear has `onlyUpgrade`. I investigated those items, but I cannot find something that could discriminate from not upgradable items. If you found, please share me and I'll add the condition.
